controllers/neutronapi_controller.go

```diff
@@ -809,6 +832,71 @@ func (r *NeutronAPIReconciler) transportURLCreateOrUpdate(instance *neutronv1bet
 	return transportURL, op, err
 }
 
+// TODO(ihar) this function could live in lib-common
+// deleteExternalSecrets - delete obsolete secrets for external dataplane consumption
+func (r *NeutronAPIReconciler) deleteExternalSecrets(
+	ctx context.Context,
+	h *helper.Helper,
+	instance *neutronv1beta1.NeutronAPI,
+) error {
+	cm := &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("%s-external-config", instance.Name),
```

Comment:
decision to make: whether to have a single "external-config" for all EDP services, or one per service.

Reply:
I will have one secret per agent